src/tests/JIT/opt/Devirtualization/Comparer_get_Default.cs

```diff
@@ -271,33 +271,36 @@ public static int TestEntryPoint()
                 Compare_Struct1_Nullable(null, null);
                 Compare_Struct2_Nullable(null, null);
 
-                // workaround for: https://github.com/dotnet/roslyn/issues/68770
-                static T Bitcast<T>(long l) => Unsafe.As<long, T>(ref l);
-
-                var enumCharA = Bitcast<CharEnum>(a);
-                var enumCharB = Bitcast<CharEnum>(b);
-                Compare_Generic_Enum(enumCharA, enumCharB);
-
-                var enumBoolA = Bitcast<BoolEnum>(a);
-                var enumBoolB = Bitcast<BoolEnum>(b);
-                Compare_Generic_Enum(enumBoolA, enumBoolB);
-
-                var enumFloatA = Bitcast<FloatEnum>(a);
-                var enumFloatB = Bitcast<FloatEnum>(b);
-                Compare_Generic_Enum(enumFloatA, enumFloatB);
-
-                var enumDoubleA = Bitcast<DoubleEnum>(a);
-                var enumDoubleB = Bitcast<DoubleEnum>(b);
-                Compare_Generic_Enum(enumDoubleA, enumDoubleB);
-                Compare_Double_Enum(enumDoubleA, enumDoubleB);
-
-                var enumIntPtrA = Bitcast<IntPtrEnum>(a);
-                var enumIntPtrB = Bitcast<IntPtrEnum>(b);
-                Compare_Generic_Enum(enumIntPtrA, enumIntPtrB);
-
-                var enumUIntPtrA = Bitcast<UIntPtrEnum>(a);
-                var enumUIntPtrB = Bitcast<UIntPtrEnum>(b);
-                Compare_Generic_Enum(enumUIntPtrA, enumUIntPtrB);
+                if (!TestLibrary.Utilities.IsNativeAot)
```

Comment:
Make this `PlatformDetection.IsRareEnumsSupported` so that it has parity with libraries tests?